libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts

```diff
@@ -10,37 +10,37 @@ import { createDateValueFormatter } from "./dateValueFormatter";
 import { DateFormat, DEFAULT_DATE_FORMAT } from "./dateValueParser";
 import { createDefaultDateFormatter } from "./defaultDateFormatter";
 
-export function findDateAttributeUri(dimensions: IDimensionDescriptor[]): string | undefined {
-    const dateAttrDescriptor = dimensions
+export function findDateAttributeUris(dimensions: IDimensionDescriptor[]): string[] {
+    const dateAttrsDescriptor = dimensions
         .reduce((attrDimensions: IAttributeDescriptor[], dimensionItem: IDimensionDescriptor) => {
             attrDimensions.push(...dimensionItem.headers.filter(isAttributeDescriptor));
             return attrDimensions;
         }, [])
-        .find(
+        .filter(
             (attrDescriptor: IAttributeDescriptor): boolean =>
                 attrDescriptor.attributeHeader.type === "GDC.time.day_us",
         );
-    if (!dateAttrDescriptor) {
-        return undefined;
-    }
-    return dateAttrDescriptor.attributeHeader.formOf.uri;
+    return dateAttrsDescriptor.map((dateAttrDescriptor) => dateAttrDescriptor.attributeHeader.formOf.uri);
 }
 
 export function transformDateFormat(
     resultHeader: IResultHeader,
-    dateAttributeUri?: string,
+    dateAttributeUris?: string[],
     dateFormat?: DateFormat,
 ): IResultHeader {
     if (
         !isResultAttributeHeader(resultHeader) ||
-        !dateAttributeUri ||
+        !dateAttributeUris?.length ||
         !dateFormat ||
         dateFormat === DEFAULT_DATE_FORMAT
     ) {
         return resultHeader;
     }
     const resultHeaderUri = resultHeader.attributeHeaderItem.uri;
-    if (!resultHeaderUri.startsWith(dateAttributeUri)) {
+    const foundUri = dateAttributeUris.find((dateAttributeUri) =>
```

Comment:
Nitpick: `some` might be more appropriate here, you never use the actual value, you only check if any element meets the criteria.

Reply:
Done